README.md

```diff
@@ -108,24 +131,85 @@ if($player instanceof Player && $player->isOnline()){
 
 - Target
 
+After setting the camera to free mode, you need to explicitly assign a target.  
+This allows the camera to visually track a specific entity.  
+Unlike SetActorLinkPacket, it does not follow the entity automatically.  
```

Comment:
This sentence is simply confusing. It does follow the entity automatically, you just have to instruct it correctly

Reply:
Perhaps I should clarify that it is perspective tracking